src/github.com/getlantern/flashlight/logging/logging_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/getlantern/go-loggly"
 	"github.com/getlantern/golog"
-	"github.com/stretchr/testify/assert"
+	"github.com/getlantern/testify/assert"
```

Comment:
That's been superseded by this - https://github.com/getlantern/lantern/issues/3851